app/models/app.rb

```diff
@@ -19,9 +19,9 @@ class App
   # Some legacy apps may have string as key instead of BSON::ObjectID
   # identity :type => String
   field :_id,
-    type: String,
-    pre_processed: true,
-    default: ->{ BSON::ObjectId.new.to_s }
+        type: String,
+        pre_processed: true,
+        default: ->{ BSON::ObjectId.new.to_s }
```

Comment:
You're right, this doesn't look too hot just yet.

There's a tweakable setting called `Style/AlignParameters`: https://github.com/bbatsov/rubocop/blob/2c9d207af5ffb639d32f17a97251956f16c2356b/config/default.yml#L135-L152 Your description matches the mode `with_fixed_indentation`, giving that a try.